expr/tagquery/tag.go

```diff
@@ -13,6 +13,8 @@ import (
 //go:generate msgp
 type Tags []Tag
 
+//Parse a list of key value pairs (not including the metric name) into a list of Tags
```

Comment:
`ParseTags()` may also be used in places where the metric name is specified as a tag, like `name=some.metric.name`

Reply:
Updated in ea8dbb9af8c9bbf4eeedf564c71df0275f6ae51c, parenthetical isn't necessary with the examples.